spec/views/orders/index.html.erb_spec.rb

```diff
@@ -28,7 +28,8 @@
     ])
   end
 
-  it "renders a list of orders" do
```

Comment:
To include page-specific JavaScript asset, use `yield :javascript` or `yield :head` in application layout and `content_for` on the page.
Don't use public folder for assets in a direct way also.